src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super("circle", color);
```

Comment:
I don't think we need to pass String "circle" in constructor. You can use getClass().getSimpleName() methods to get name of the class

Reply:
We cannot use **super(getClass().getSimpleName(), color);** because of **"Cannot reference 'Object.getClass' before supertype constructor has been called"**